src/encoded/tests/data/inserts/pipeline.json

```diff
@@ -178,5 +178,16 @@
         "lab": "brenton-graveley",
         "award": "U54HG007005",
         "uuid": "cd340a00-50aa-11e4-916c-0800200c9a66"
+    },
+    {
+        "accession": "ENCPL001NAT",
+        "analysis_steps": [
+            "align-star-step-v-1"
+        ],
+        "status": "released",
+        "title": "RBNS pipeline",
+        "lab": "brenton-graveley",
+        "award": "U54HG007005",
+        "uuid": "d71cee3c-1d49-46f1-b4d6-45e0848a0f33"
```

Comment:
have you created this insert to show that the audit will appear on the local demo? If that is the reason I don't think you should do it. You should definitely write a test for the audit triggering in the correct circumstances, but you should not create "wrong" inserts for the sake of audit triggering.

Reply:
Will remove the insert